ui/pages/confirmations/components/advanced-gas-fee-popover/advanced-gas-fee-gas-limit/advanced-gas-fee-gas-limit.test.js

```diff
@@ -154,7 +154,7 @@ describe('AdvancedGasFeeGasLimit', () => {
     );
     expect(
       screen.queryByText(
-        `Gas limit must be greater than 29999 and less than 7920050`,
+        `Gas limit must be greater than 29999 and less than 30000000`,
```

Comment:
isn't the lower limit 20999?

Reply:
yeah I also tried to fix that and got stuck until I realised that we have an error logic where we use `editGasLimitOutOfBoundsV2` which accepts two values so in the mocking data we are sending originalGasEstimate: '0x78D9B2' `(30000)` and there is this logic [here](https://github.com/MetaMask/metamask-extension/blob/main/ui/pages/confirmations/components/advanced-gas-fee-popover/advanced-gas-fee-gas-limit/advanced-gas-fee-gas-limit.js#L77) changing the min to `29999`.